Dockerfile

```diff
@@ -0,0 +1,27 @@
+##
+## build with:
+## docker buildx build --platform=local --output dist .
+##
+## binary for your arch will be inside dist folder
+##
+## to build for OSX, run previously:
+## docker buildx create --use --platform darwin/amd64
+##
+
+FROM --platform=$BUILDPLATFORM tonistiigi/xx:golang AS xgo
+
+FROM --platform=$BUILDPLATFORM golang:1.13-alpine AS build
+
+ENV CGO_ENABLED=0
+COPY --from=xgo / /
+
+ARG TARGETPLATFORM
+RUN go env
+
+WORKDIR /src
+COPY . /src/
+
+RUN go build -o /go/bin/nextdns
```

Comment:
You may want to add `-ldflags="-s -w"` as we do for binary distrib.

Reply:
I'm not familiar with this

Reply:
It strips the binary to make it smaller. Check this post for more info: https://blog.filippo.io/shrink-your-go-binaries-with-this-one-weird-trick/

Reply:
added